matrix-meetings-bot/test/model/ExternalData.test.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2023 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ExternalData,
+  extractOpenXChangeExternalReference,
+} from '../../src/model/ExternalData';
+
+function createEvent(opts: {
+  folder: string | number;
+  id: string | number | undefined;
+  rrules: any;
+}): ExternalData {
+  return {
+    'io.ox': {
+      folder: opts.folder,
+      id: opts.id,
+      rrules: opts.rrules,
+    },
+  };
+}
+
+describe('extractOpenXChangeExternalReference', () => {
+  it('should return reference', () => {
+    expect(
+      extractOpenXChangeExternalReference(
+        createEvent({
+          folder: 'cal://0/31',
+          id: 'cal://0/31.1.0',
+          rrules: ['FREQ=DAILY;COUNT=10'],
+        })
```

Comment:
Not sure if we really need a function for it. It would have the same length but without the indirection:

```suggestion
        'io.ox': {
          folder: 'cal://0/31',
          id: 'cal://0/31.1.0',
          rrules: ['FREQ=DAILY;COUNT=10'],
        }
```

Reply:
sounds fine to me, changed the code to use 'io.ox', function removed